### ISSUE TITLE
[Vue2] Uncaught TypeError: Super expression must either be null or a function.

### ISSUE THREAD

**Issue by ekoooo** [hrynko/vue-pdf-embed#119]
When i install the latest version `vue-pdf-embed@1.1.5` in `vue2`, works normal in development mode, but build mode i got an error in browser.

```text
Uncaught TypeError: Super expression must either be null or a function
    at chunk-libs.7e877d2c.js:8:175126
    at chunk-libs.7e877d2c.js:8:175194
    at Object.defineProperty.value (chunk-libs.7e877d2c.js:8:179400)
    at __w_pdfjs_require__ (chunk-libs.7e877d2c.js:8:369384)
    at Object.defineProperty.value (chunk-libs.7e877d2c.js:8:122091)
    at __w_pdfjs_require__ (chunk-libs.7e877d2c.js:8:369384)
    at chunk-libs.7e877d2c.js:8:373178
    at chunk-libs.7e877d2c.js:8:373765
    at factory (chunk-libs.7e877d2c.js:8:373789)
    at 783 (chunk-libs.7e877d2c.js:8:373808)
```

I tried lowering the version one by one, until `vue-pdf-embed@1.1.0` version didn’t have this error.

**Comment by hrynko** [comment]
Hi @ekoooo,

Could you provide more details about your environment (OS, browser versions, etc.)?

**Comment by zccmynotebook** [comment]
I also meet this question;["vue": "^2.5.17", "webpack": "^4.19.0",] it is not caused by browser, I think it due to webpack or something else; as it is ok in local server without being compressed or compacted. thanks for your attention.

**Comment by hrynko** [comment]
Should be resolved in `v2.0.0`, otherwise please reopen the issue
